### DISCUSSION TITLE
2.0.0 (2023-07-28)

### DISCUSSION THREAD

**Discussion by brendanhay** [brendanhay/amazonka#944] [Announcements]
The Amazonka team would like to announce the full `2.0.0` release now available on [Hackage](https://hackage.haskell.org/package/amazonka). There's a lot of work in here, many thanks to all the contributors past and present for getting this over the line. With this release finally out the door we hope to return to a healthy cadence of Hackage releases.

For those still using the `1.6.1` version from Hackage, many things have changed. If you haven't been following along with the release candidate process some suggested reading is the [migration guide](https://github.com/brendanhay/amazonka/blob/main/lib/amazonka/README.md#migrating-from-161-to-20) for an overview and the [CHANGELOG](https://github.com/brendanhay/amazonka/blob/main/lib/amazonka/CHANGELOG.md) for full details.

Please file any feedback under the related [GitHub discussion](https://github.com/brendanhay/amazonka/discussions/944) or create a [new issue](https://github.com/brendanhay/amazonka/issues).

**Comment by RobertFischer** [comment]
Yeah!  So awesome!

**Comment by endgame** [comment]
@brendanhay Thank you for trusting me with the commit bit. It's a great feeling to get this over the line.

**Reply by brendanhay** [reply]
Thanks for driving everything forward!

**Comment by arianvp** [comment]
Amazing. Thanks for all the work everybody!

**Comment by mariatsji** [comment]
Amazing work, congratulations!

**Comment by ulidtko** [comment]
Nice. Awesome! :raised_hands:

**Comment by BharatKJain** [comment]
Thank you for this! We can finally enable OIDC with EKS by using web-identity support. 🙌

**Comment by andys8** [comment]
Great to see 2.0.0
